service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/FeeServiceTest.java

```diff
@@ -87,15 +87,15 @@ void shouldGetAFeeResponseWhenGivenAPopulated() {
 
             List<FeeResponse> fees = feeService.getFees(List.of(CARE_ORDER, OTHER, PLACEMENT));
 
-            assertThat(fees).hasSize(3);
-            assertThat(fees).containsOnly(careOrderResponse, otherResponse, placementResponse);
+            assertThat(fees).hasSize(3)
```

Comment:
hasSize should not be needed

Reply:
good point, `containsOnly` acts as the size check